python/grass/jupyter/display.py

```diff
@@ -53,6 +53,28 @@ def run(self, module, **kwargs):
         else:
             raise ValueError("Module must begin with letter 'd'.")
 
+    def __getattr__(self, attr, **kwargs):
+        """Parse attribute to GRASS display module. Attribute should be in
+        the form 'd_{module}'."""
```

Comment:
This text may use some improvement, but more importantly, add examples to the class docstring.

```
>>> m = ...
>>> m.d_vect(...)
````

Reply:
I really meant docstring of the class, not the one for the method. `__getattr__` is too special for it to be an obvious place to look for documentation of `m.d_vect` behavior.